src/mumble/ListenerVolumeSlider.cpp

```diff
@@ -39,18 +43,42 @@ void ListenerVolumeSlider::on_VolumeSlider_sliderReleased() {
 
 	if (handler->m_version >= Mumble::Protocol::PROTOBUF_INTRODUCTION_VERSION) {
 		// With the new audio protocol, volume adjustments for listeners are handled on the server and thus we want
-		// to avoid spamming updates to the adjustments, which is why we only update them once the slider is released.
-		MumbleProto::UserState mpus;
-		mpus.set_session(Global::get().uiSession);
+		// to avoid spamming updates to the adjustments, which is why we only update them with a delay.
+
+		if (m_channelBuffer == m_channel->iId && m_adjustmentBuffer == adjustment) {
+			return;
+		}
 
-		MumbleProto::UserState::VolumeAdjustment *adjustmentMsg = mpus.add_listening_volume_adjustment();
-		adjustmentMsg->set_listening_channel(m_channel->iId);
-		adjustmentMsg->set_volume_adjustment(adjustment.factor);
+		m_channelBuffer    = m_channel->iId;
+		m_adjustmentBuffer = adjustment;
+
+		// Timer values: 0, 50, 150, 350, 750, 1000 (ms)
+		m_resetTimer.stop();
+		m_sendTimer.start(m_currentTimer);
+		m_currentTimer = std::min(1000u, (m_currentTimer + 25) * 2);
 
-		handler->sendMessage(mpus);
 	} else {
 		// Before the new audio protocol, volume adjustments for listeners are handled locally
 		Global::get().channelListenerManager->setListenerVolumeAdjustment(Global::get().uiSession, m_channel->iId,
 																		  adjustment);
 	}
 }
+
+void ListenerVolumeSlider::sendToServer() {
+	ServerHandlerPtr handler = Global::get().sh;
+
+	m_resetTimer.start(3000);
```

Comment:
I think 2s would probably also be sufficient. But I'm also fine with leaving this at 3s.

Reply:
Mh, I have excessively tested all the timings for 3s now. And I really do not feel like going over that all over again with 2s :/